e2e/pageobjects/openshift/OpenShiftConsole4x.ts

```diff
@@ -0,0 +1,164 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import { injectable, inject } from 'inversify';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { CLASSES } from '../../inversify.types';
+import { TestConstants } from '../../TestConstants';
+import { By, Key } from 'selenium-webdriver';
+
+@injectable()
+export class OpenShiftConsole4x {
+
+    private static readonly CHE_OPERATOR_LOGO_NAME: string = '//h1[contains(@class, \'logo__name__clusterserviceversion\') and text()=\'Eclipse Che\']';
+    private static readonly ECLIPSE_CHE_PREFIX_URL: string = 'che-';
+
+    constructor(
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async waitNavpanelOpenShift () {
+        await this.driverHelper.waitVisibility(By.xpath('//nav[@class=\'pf-c-nav\']'));
```

Comment:
Just note. I agree with the approach to inline locators in the methods if they used just in this method. But what about to declare it by dedicated constant with the appropriate and understandable name?
```
    async waitNavpanelOpenShift () {
        const navPanelLocator: By = By.xpath('//nav[@class=\'pf-c-nav\']');

        await this.driverHelper.waitVisibility(navPanelLocator);
    }
```

Reply:
I don't mind, it seems, it will do a code more understanding..
It will reworked.